src/Components/FormCreation/CreateEditForm.js

```diff
@@ -662,40 +663,20 @@ function CreateEditForm({ programId }) {
                 horizontal: "left"
               }}
             />
-            <DragDropContext onDragEnd={onDragEnd}>
-              {sections &&
-                sections.map((section, key) => (
-                  <div key={uniquekey + section._id}>
-                    <FormWrapper key={section._id} id={"section_" + key}>
-                      <FormSection
-                        key={key + "_section"}
-                        numSections={sections.length}
-                        sectionNum={key + 1}
-                        sectionData={section}
-                        updateActiveSection={updateActiveSection}
-                        active={activeSection === key}
-                        handleAddSection={handleAddSection}
-                        handleTitleUpdate={handleTitleUpdate}
-                        handleDescriptionUpdate={handleDescriptionUpdate}
-                        handleSectionTypeUpdate={handleSectionTypeUpdate}
-                        handleMoveSection={handleMoveSection}
-                        handleDeleteSection={handleDeleteSection}
-                        setShowMoveSectionsDialog={setShowMoveSectionsDialog}
-                      />
-                    </FormWrapper>
-                  </div>
-                ))}
-            </DragDropContext>
```

Comment:
On first code pass, seems a bit weird this is being deleted? Will see what it looks like in app after review.

Reply:
It was copy and pasted by Guy for the delete section functionality. When the section was deleted, but still able to be undone this code executed and showed double the form.